WeakAurasOptions/Cache.lua

```diff
@@ -126,20 +126,13 @@ local function Lev(str1, str2)
 end
 
 function spellCache.BestKeyMatch(nearkey)
-  for key, value in pairs(cache) do
-    if(nearkey == key) then
-      return key;
-    end
-  end
-  for key, value in pairs(cache) do
-    if(nearkey:lower() == key:lower()) then
-      return key;
-    end
-  end
   local bestKey = "";
   local bestDistance = math.huge;
   local partialMatches = {};
   for key, value in pairs(cache) do
+    if(nearkey:lower() == key:lower()) then
```

Comment:
That's not the same algorithm anymore.

Reply:
What specifically breaks, though?

Reply:
The old algorithm prefer matches with the same case to matches without the same. E..g if there's a all upper case match, and that one is before the correct match, then this will return the key uppercased.

Reply:
Hmm, I see.